src/sarif_converter.ts

```diff
@@ -81,8 +81,9 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
       },
       properties: {
         tags: [language, ...suggestion.tags, ...suggestion.categories],
+        exampleCommitFixes: suggestion.exampleCommitFixes,
         precision: 'very-high',
-      } as { tags: string[]; precision: string; cwe?: string[] },
+      } as { tags: string[]; exampleCommitFixes?: ExampleCommitFix[]; precision: string; cwe?: string[] },
```

Comment:
question: maybe we should have an interface for `properties`?

Reply:
I don't hate this idea either, just be careful as properties are used in many places and should each have a different interface.

Reply:
Done